Gruntfile.js

```diff
@@ -54,6 +57,7 @@ module.exports = function(grunt) {
         'styles',
         'minify',
         'assets:library',
+        'schematics',
```

Comment:
I noticed that the schematics were not included in the Artifactory package. There is a separate, quite similar grunt task named "compile" which is run when building both library and docs (npm run build). This would need to be added to that task as well.